### ISSUE TITLE
Branch indicators lack contrast

### ISSUE THREAD

**Issue by marijnvdwerf** [decompme/decomp.me#757]
In light mode, most branch indicators don't contrast enough with the background.

Also, the colours seem pretty basic (`#fof` and `#0ff`) and all over the place, it might be interesting to look into adopting some predefined palette. Some common datavis palettes can be found at https://plotly.com/python/discrete-color/#color-sequences-in-plotly-express.

Example:
<img width="291" alt="image" src="https://github.com/decompme/decomp.me/assets/737603/509c930e-2b0c-43b1-bb61-5529ca4a570c">

Constrast values: (counting everything below 3.0 as a failure, though this should probably be more like 4.5)
<img width="312" alt="image" src="https://github.com/decompme/decomp.me/assets/737603/f5c81e94-48e5-4b90-904d-ad170a51aa32">

**Comment by mkst** [comment]
Is this a duplicate of #https://github.com/decompme/decomp.me/issues/678 ?
